light-curve/light_curve/light_curve_py/features/rainbow/temperature.py

```diff
@@ -0,0 +1,176 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import Dict, List
+
+import numpy as np
+
+__all__ = ["temperature_terms", "BaseTemperatureTerm", "ConstantTemperatureTerm", "SigmoidTemperatureTerm"]
+
+
+@dataclass()
+class BaseTemperatureTerm:
+    """Bolometric term for the Rainbow"""
+
+    @staticmethod
+    @abstractmethod
+    def parameter_names() -> List[str]:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def parameter_scalings() -> List[str]:
+        """Describes how to unscale the parameters - like time, timescale, flux or do not scale"""
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def value(self, t, params) -> float:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def initial_guesses(t, m, sigma, band) -> Dict[str, float]:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def limits(t, m, sigma, band) -> Dict[str, float]:
+        return NotImplementedError
+
+
+@dataclass
+class ConstantTemperatureTerm(BaseTemperatureTerm):
+    """Constant temperature"""
+
+    @staticmethod
+    def parameter_names():
+        return ["T"]
+
+    @staticmethod
+    def parameter_scalings():
+        return [None]
+
+    @staticmethod
+    def value(t, T):
```

Comment:
These variable names are ambiguous. Let's use `temp` or something like that for temperature, code-wise?

It doesn't apply to the minuit parameter name, it is up to you keep it "T" or change to "Temp"

Reply:
done